tests/function.test.ts

```diff
@@ -50,10 +51,10 @@ describe('NodejsFunction tests', () => {
 
     it.each`
       handler                   | entry
-      ${null}                   | ${'index.ts'}
-      ${'source/index.handler'} | ${'source/index.ts'}
-      ${'main.func'}            | ${'main.ts'}
-      ${'a/b/c.h'}              | ${'a/b/c.ts'}
+      ${null}                   | ${join(findProjectRoot() ?? '', 'index.ts')}
```

Comment:
Hey @stephankaag , thank you for the contribution
Could you please make a small fix: use `process.cwd()` instead of `findProjectRoot()`, so you can be sure that it will return the current project folder according to `mock-fs` documentation, and `findProjectRoot` is not supposed to be used to find `this` project root folder, but it supposed to find a user's project root folder.
thank you